.github/workflows/cp-pipeline-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Pipeline Manager
+
+on:
+  workflow_call:
+
+permissions:
+  contents: write # to be able to publish a GitHub release
+  issues: write # to be able to comment on released issues
+  pull-requests: write # to be able to comment on released pull requests
+  packages: write # Required for pushing images  
+
+jobs:
+  pipeline-manager:
+    name: Pipeline Manager
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Print all ENVS
```

Comment:
@visoedhwa @arolos does this still need to be in the final version?

Reply:
oh, that needs to be removed

Reply:
Print all ENVS needs to be removed